docs/guidelines/platforms/android/guideline_operable_android.md

```diff
@@ -282,6 +304,77 @@ The example given in the **Screenshot 4.** - **Avoid using ClickableSpan** and i
 
 ---
 
+### Heading and Labels (WCAG 2.4.6 - Level AA)
+
+Based on this guideline, users should clearly understand the purpose of the heading or label. It should be descriptive and give information about the content that comes next.
+
+> This guideline covers point *2.4.6 Headings and Labels - Level AA of the WCAG standard.*
+
+#### ✅ Success technique(s)
+
+When the user chooses to navigate between headings instead of between paragraphs or between word, make sure that sections on screen are defined as headings so that users can "skim" through them to locate the specific content they need.
+
+For Views, you can set the `android:accessibilityHeading` attribute to `true` for a view to be treated as a heading (requires minSdk >= 28). Alternatively, you can set `ViewCompat.setAccessibilityHeading(view, true)` for older versions.
```

Comment:
Saw this comment in an e-mail, so I just wanted to chime in since I was just recently writing up the Bypass blocks section. :D

This seems like a much more appropriate chapter to go into details regarding headings to me, I can remove it from Bypass blocks and just reference this chapter. There we can have all the info about navigation through headings (e.g. how to enable it and use it), and then just reference this chapter for implementation details. How does that sound?

Reply:
Having the details about headings in here seems more intuitive to me, as the bypass blocks section seems more like something that should give an overview, but not necessarily go into details for each option. I'd go with @sttihan's suggestion.

Reply:
@sttihan since you also worked on headings, please feel free to suggest any additional topics or details that may not have been covered in this section 😄

Reply:
The only extra thing that we had there was a bit bigger example, with some showcase input fields between the headings + an example of how to set the header programmatically. I ended up removing it and linking to this, but I think I still have it stashed somewhere, so I can send you. :D

However, to be completely honest, I'm not sure it is even needed, since headings are quite straightforward to implement and I think the information that is currently listed here is enough. I think it's not bad to have a complete example for setting the header programmatically, so that people can copy-paste it to their projects, but for the other stuff, it might be unnecessary. But if you feel like the chapter could benefit from it, they could be added.

Reply:
@sttihan I've[ added an additional example](https://github.com/infinum/accessibility-mobile-standards/pull/55/commits/623fa558b1c247e49649da3a632e374b4ed81353) which utilises `AccessibilityDelegateCompat`, as I think that should be beneficial too